community/qemu/build

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
+
+patch -p0 < 001-map-sync.patch
+patch -p0 < 002-ppc.patch
+patch -p0 < 003-sigtrm-max.patch
+patch -p0 < 004-syscall.patch
+patch -p0 < 005-elfload.patch
+
+./configure \
+    --prefix=/usr \
+    --localstatedir=/var \
+    --sysconfdir=/etc \
+    --python=/usr/bin/python3 \
+    --disable-glusterfs \
+    --disable-debug-info \
+    --disable-bsd-user \
+    --disable-werror \
+    --disable-xen \
+    --enable-kvm \
+    --enable-lzo \
```

Comment:
Is this vendored in or is it a needed dependency? (Assuming this is the compression lib)